src/main/java/org/example/Bosh.java

```diff
@@ -0,0 +1,316 @@
+                        for (int i = 0; i < drinksCount; i++) {
+                            cleanCheck();
+                            ingredientsCheck();
+                            count += 1;
+                            makeCoffee(Coffee.ESPRESSO);
+                        }
+                        System.out.println(Coffee.ESPRESSO + " готов в кол-ве: " + drinksCount + "шт!\n");
+                        logger.log(Level.INFO, "ESPRESSO has been made");
+                    }
+                    case 2 -> {
+                        System.out.println("Выбран напиток: " + Coffee.CAPPUCCINO + "\n");
+                        System.out.println("Введите кол-во напитков: ");
+                        int drinksCount = in.nextShort();
+                        for (int i = 0; i < drinksCount; i++) {
+                            cleanCheck();
+                            ingredientsCheck();
+                            count += 1;
+                            makeCoffee(Coffee.CAPPUCCINO);
+                        }
+                        System.out.println(Coffee.CAPPUCCINO + " готов в кол-ве: " + drinksCount + "шт!\n");
+                        logger.log(Level.INFO, "CAPPUCCINO has been made");
+                    }
+                    case 3 -> {
+                        System.out.println("Выбран напиток: " + Coffee.AMERICANO + "\n");
+                        System.out.println("Введите кол-во напитков: ");
+                        int drinksCount = in.nextShort();
+                        for (int i = 0; i < drinksCount; i++) {
+                            cleanCheck();
+                            ingredientsCheck();
+                            count += 1;
+                            makeCoffee(Coffee.AMERICANO);
+                        }
+                        System.out.println(Coffee.AMERICANO + " готов в кол-ве: " + drinksCount + "шт!\n");
+                        logger.log(Level.INFO, "AMERICANO has been made");
+                    }
+                    case 4 -> {
+                        System.out.println("Выбран напиток: " + Coffee.LATTE + "\n");
+                        System.out.println("Введите кол-во напитков: ");
+                        int drinksCount = in.nextShort();
+                        for (int i = 0; i < drinksCount; i++) {
+                            cleanCheck();
+                            ingredientsCheck();
+                            count += 1;
+                            makeCoffee(Coffee.LATTE);
+                        }
+                        System.out.println(Coffee.LATTE + " готов в кол-ве: " + drinksCount + "шт!\n");
+                        logger.log(Level.INFO, "LATTE has been made");
+                    }
+                }
+            } catch (Exception ignored) {
+            }
+            logger.log(Level.INFO, "Looked throughout the menu");
+        }
+    }
+
+    public void ingredientsCheck() throws Exception {
```

Comment:
```suggestion
    public void checkIngredients() throws Exception {
```
называем методы по конвенции Java